mcux/hal_nxp.cmake

```diff
@@ -263,7 +263,7 @@ endif()
 if(${MCUX_DEVICE} MATCHES "MIMXRT(5|6)")
   set(CONFIG_USE_driver_pca9420 true)
```

Comment:
@VitekST commit message should contain a prefix like this "mcux: hal_nxp: ".

Thus your commit message should be "mcux: hal_nxp: Fix redundant whitespace".

Reply:
I'll squash the commits instead, as fixup commits, as I found out, are not viewed kindly on the main repo. Thanks for letting me know.

Reply:
OK.